src/pages/plugins/index.md.ejs

```diff
@@ -2,17 +2,46 @@
 layout: content.html
 ---
 
+<%
+const require = this.process.mainModule.require
```

Comment:
interesting - did you copy this from my ancient code?

Reply:
Indeed - require wasn't being recognised by EJS without it. Moot point, as relevant data has been moved to inside file